go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/imjoey/terraform-provider-ovirt
+
+go 1.12
+
+require (
+	github.com/hashicorp/terraform v0.12.0
+	github.com/ovirt/go-ovirt v4.0.7-0.20190516061919-494f09a2cd4c+incompatible
```

Comment:
@imjoey Is it possible to use "v" before your tag on https://github.com/oVirt/ovirt-engine-sdk-go/blob/master/automation/deploy-to-go-ovirt.sh#L71 ?

`go get github.com/ovirt/go-ovirt` grab the last version (v4.0.6).

Reply:
@llomgui , Sure. I'll fix this for better fit for `go modules` right now.

Reply:
@imjoey Tell me once you re-tagged versions, I will commit again.

Reply:
Hi @llomgui, `v4.3.4` has been just tagged for `go get github.com/ovirt/go-ovirt`.